spec/models/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :email }
+    it { is_expected.to have_db_column :encrypted_password }
+  end
+
+  it { is_expected.to have_many :invoices }
```

Comment:
please put this test in a describe 'Associations' block

Reply:
done